spec/requests/admin_payment_confirmation_report_upload_spec.rb

```diff
@@ -57,6 +63,23 @@
         end
       end
 
+      context "the CSV has invalid data" do
```

Comment:
Could we can test the not queueing part in one of the existing contexts rather than creating a new one?